### PR TITLE
Route53

### PR DESCRIPTION
**Pull request by Faria-Marcio**
# Description

Exercise: C03-AWS03

Student Name: Marcio de Faria

> Creation of Route53 private hosted zone to access my private instances.

## Type of change

- [ X ] Exercise Submission (one exercise per PR)
- [ X ] This PR is just for **one** exercise of a class
- [ X ] PR name follows the pattern `<github-username>/<exercise-number>`
- [ X ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
- [ X ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-aws03/Faria-Marcio/c03-aws03.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	Type: A - IPv4 address
+	Alias: Yes
+	Alias Target: dualstack.internal-alb-faria-197225106.ap-southeast-2.elb.amazonaws.com.
+
+
+
+#Post here any findings, how option you decided to go with, records created, domain created, and anything else that you find interesting:
+
+   - As I had deleted my NAT gateways, Load Balancers, and Auto Scalling Groups, I needed to recreate them and well as reconfigure my Route Table to use the new NGW.
+   - The trickest part was to find out that I needed to enable DNS hostnames in my VPC. It took my a while to realize that.
```
Comment (caiocezart) [inline classes/03class/exercises/c03-aws03/Faria-Marcio/c03-aws03.txt]: good finding.. but this usually is ON by default

File: classes/03class/exercises/c03-aws03/Faria-Marcio/c03-aws03.txt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+
+#Result:
+
+   --> I connected in my public EC2 instance and through it I accessed the private route53 hostname for my private instances:
+	ubuntu@ip-192-168-101-164:~$ curb.amazonaws.comfaria-197225106.ap-southeast-2.elb
```
Comment (caiocezart) [inline classes/03class/exercises/c03-aws03/Faria-Marcio/c03-aws03.txt]: I think you have messed with ctrl + v here.. can you update the text and push again?

File: classes/03class/exercises/c03-aws03/Faria-Marcio/c03-aws03.txt (hunk 1 above)
Comment (Faria-Marcio) [reply classes/03class/exercises/c03-aws03/Faria-Marcio/c03-aws03.txt]: Yeah, I don't know why it was set disabled.

Comment (denstorti) [inline classes/03class/exercises/c03-aws03/Faria-Marcio/c03-aws03.txt:22]: Good stuff.

**Pull request merged**